src/GeneralStateTestsFiller/stCallCodes/EOF1_ExecutionFiller.yml

```diff
@@ -0,0 +1,1375 @@
+      # JUMP inside code section
+      # code section: 'jump x sstore(0, 1) x: sstore(0, 2) stop()'
+      code: ':raw 0xEF000101000F0060085660016000555B600260005500'
+      nonce: 0
+      storage: {}
+
+    114f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      # JUMP inside code section with data section
+      # code section: 'jump x sstore(0, 1) x: sstore(0, 2) stop()'
+      # data section: 'aabbccdd'
+      code: ':raw 0xEF000101000F0200040060085660016000555B600260005500AABBCCDD'
+      nonce: 0
+      storage: {}
+
+    124f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      # JUMPI inside code section
+      # code section: 'push1(1) jumpi x sstore(0, 1) x: sstore(0, 2) stop()'
+      code: ':raw 0xEF0001010011006001600A5760016000555B600260005500'
+      nonce: 0
+      storage: {}
+
+    134f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      # JUMPI inside code section with data section
+      # code section: 'push1(1) jumpi x sstore(0, 1) x: sstore(0, 2) stop()'
+      # data section: 'aabbccdd'
+      code: ':raw 0xEF0001010011020004006001600A5760016000555B600260005500AABBCCDD'
+      nonce: 0
+      storage: {}
+
+    144f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      # JUMP destination as last byte before terminating instruction
+      # code section: 'sstore(0, 1) jump x sstore(0, 2) stop() x: stop()'
+      code: ':raw 0xEF000101000F006001600055600D5660026000555B00'
+      nonce: 0
+      storage: {}
+
+    154f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      # JUMPI destination as last byte before terminating instruction
+      # code section: 'sstore(0, 1) push1(1) jumpi x sstore(0, 2) x: stop()'
+      code: ':raw 0xEF00010100110060016000556001600F5760026000555B00'
+      nonce: 0
+      storage: {}
+
+    164f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      # Data section is 0x65 (101) bytes long, so header contains 0x65 (PUSH6) byte,
+      # but it must not affect jumpdest analysis (JUMPDEST stays valid)
+      # code section: 'jump x invalid x: sstore(0, 1) stop()'
+      code: ':raw 0xEF000101000B02006500600456FE5B600160005500FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF'
+      nonce: 0
+      storage: {}
+
+    174f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      # Data section is 0x67 (103) bytes long, so header contains 0x67 (PUSH8) byte,
+      # but it must not affect jumpdest analysis (JUMPDEST stays valid)
+      # code section: 'push1(1) jumpi x invalid x: sstore(0, 1) stop()'
+      code: ':raw 0xEF000101000D020067006001600657FE5B600160005500FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF'
```

Comment:
This code is invalid: the data section is longer by 2 bytes from expected, remove `FFFF`.

Reply:
Fixed.